Description: Relax dependencies version
 Relax dependencies version to match the version in Debian.
Author: Yogeswaran Umasankar <yogu@debian.org>
Forwarded: not-needed
Last-Update: 2024-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@ path = "src/lib.rs"
 version = "0.5.0"
 
 [dependencies.nalgebra]
-version = "0.32"
+version = "0.33"
 features = ["rand"]
 
 [dependencies.num-traits]
